review-notifications/src/Github.js

```diff
@@ -128,6 +131,22 @@ class Github extends Component {
     });
   }
 
+  readMoreAndLess() {
+    var dots = document.getElementById('dots');
```

Comment:
these operations should be avoided in react unless there is a very specific reason to do them. If this is something you render yourself you can use refs, if it's rendered by an external library perhaps you should think about rewriting it yourselves since it seems very limited. We can discuss this more thoroughly on our next meeting since it might be a little hard through comments.

At the very least these should be `const` not `var`